utils/utils.go

```diff
@@ -43,6 +45,22 @@ func ExecCmdWithStdStreams(stdin io.Reader, stdout, stderr io.Writer, name strin
 	return nil
 }
 
+// ExecCmdReturnCode executes a command with args and returns its exit code
+func ExecCmdReturnCode(name string, args ...string) (int, error) {
+	cmd := exec.Command(name, args...)
+	err := cmd.Start()
+	if err != nil {
+		return -1, errors.Wrapf(err, "unable to execute command")
+	}
+	result := cmd.Wait()
+	if result == nil {
+		return 0, nil
+	}
+	resultError, _ := result.(*exec.ExitError)
+	returnCode := resultError.Sys().(syscall.WaitStatus)
```

Comment:
I don't think this will work if result. above returns an error code

Reply:
Agree, don't think you can discard the error above

Reply:
iirc you cannot get an error given the input I give it ... but i certainly can just error out.

Reply:
I'm a bit afraid that an access like that might segfault if the cast fails, so I think it would be better to error out gracefully

Reply:
fixed

Reply:
retracting my fix here, the _ is actually a bool we dont need.